OZprivate/rawJS/OZTreeModule/src/navigation/pinpoint.js

```diff
@@ -142,6 +142,53 @@ export function node_to_pinpoint(node) {
   ].join("")
 }
 
+/**
+ * Return a pinpoint string pointing at (node) that is stable across different versions of the OneZoom tree.
+ * Null if such a pinpoint can't be constructed. 
+ * Slower than a standard node_to_pinpoint.
+ */
+export function node_to_stable_pinpoint(node) {
```

Comment:
At some point in the future, might be interesting to consider whether we should be using this in more places where we're currently using node_to_pinpoint

Reply:
Okay cool, I've done that, and then made the highlights editor separately reject @_ozid pinpoints because I think that's the only place that really doesn't want unstable pinpoints.